php-practice/section2.php

```diff
@@ -117,6 +139,18 @@ function printNewline() {
 // HINT: Use a foreach loop
 // =======================================================================
 
+$sumArray = array(1, 2, 3, 4); 
+
+function ($sumArray) {
+	foreach ($sumArray as $value);
+	$total = 0;
+	$total += $value;
+	return ($total);
```

Comment:
Take a look at the PHP documentation, just google "PHP foreach",  and see how  it says the syntax is  supposed to look.